cli/src/generate/grammar_files.rs

```diff
@@ -243,7 +243,8 @@ pub fn generate_grammar_files(
         },
     )?;
 
-    let (_, package_json) = lookup_package_json_for_path(package_json_path_state.as_path())?;
+    let (_, package_json) = lookup_package_json_for_path(package_json_path_state.as_path())
+        .map_err(|e| generate_bindings.then(|| Err(e)).unwrap_or(Ok(())))?;
 
     // Do not create a grammar.js file in a repo with multiple language configs
     if !package_json.has_multiple_language_configs() {
```

Comment:
wouldn't that trigger the error here? you probably want to pull the `!generate_bindings` early return earlier?

Reply:
(Drive-by comment; feel free to disregard.)